### ISSUE TITLE
Fix Python 3.6 metaclass error

### ISSUE THREAD

**Issue by tfiers** [tfiers/parachute#4]
On Travis, we get the famous metaclass error:
```Python-traceback
src/parachute/__init__.py:2: in <module>
    from .validarg import *
src/parachute/validarg.py:6: in <module>
    from .validators.base import ValidatedArgument
src/parachute/validators/base.py:78: in <module>
    Generic[CanonicalParamType], metaclass=ValidatedArgumentFactory
E   TypeError: metaclass conflict: the metaclass of a derived class must be a (non-strict) 
subclass of the metaclasses of all its bases
```

**Comment by tfiers** [comment]
Fixed by d870f01e3ad13d677216f03980b913e695f9055d

**Comment by tfiers** [comment]
Fixed by d870f01
